dhis-2/dhis-web-api-test/pom.xml

```diff
@@ -21,6 +21,11 @@
         <artifactId>maven-dependency-plugin</artifactId>
         <configuration>
           <failOnWarning>true</failOnWarning>
+          <ignoredUnusedDeclaredDependencies>
+            <ignoredUnusedDeclaredDependency>com.h2database:h2</ignoredUnusedDeclaredDependency>
+            <ignoredUnusedDeclaredDependency>org.skyscreamer:jsonassert</ignoredUnusedDeclaredDependency>
+            <ignoredUnusedDeclaredDependency>com.cronutils:cron-utils</ignoredUnusedDeclaredDependency>
```

Comment:
What uses this one and does not import it?

Reply:
Added a comment in the pom explaining the issue and where it is happening.
This would probably require some more investigation, I am not sure why this is happening